source/change-streams.rst

```diff
@@ -97,6 +97,8 @@ An error is considered resumable if it meets any of the following criteria:
     * - CursorKilled
       - 237
 
+An error on the initial aggregate is not a retryable error.
```

Comment:
I think we can say that errors on *any* aggregate command (the initial or a resume attempt) are not resumeable. Only errors on a getMore command may be considered "resumable".